README.md

```diff
@@ -140,20 +140,15 @@ If your project uses Filecoin, you're in the Filecoin ecosystem! We'd love to se
 
 Following the steps below to submit your projects to be featured in this [repo](https://github.com/filecoin-project/community/tree/master/projects):
 - Create a project profile using the [project submission template](https://github.com/filecoin-project/community/blob/master/templates/project-submission-template.md). Name the file name after your project and fill in as much information as you can.  Use your project name as the file name so the community can find your project easily!
-- Create a project discussion, with the category as Show and tell - Developers  and title as `[Category] Project Name`(i.e: `[Application] Slate`), link the discussion in the How the community can engage section. This is to create a place to interact with the Filecoin community, share your updates and gain feedback on your projects!
+- Create a new discussion thread in [Discussions > Project Showcase](https://github.com/filecoin-project/community/discussions/categories/project-showcase), titled `[Category] Project Name`(i.e: `[Application] Slate`). Add this link to the "How the community can engage" section. Use this thread to share your updates, and chat with users and potential users!
```

Comment:
@mishmosh  do you think using [Community Discussion > Project Showcase] will be clearer for readers ?

Reply:
good idea, fixing